src/applications/personalization/account/components/AccountMain.jsx

```diff
@@ -72,6 +72,17 @@ class AccountMain extends React.Component {
         <AccountVerification loa={loa}/>
         {this.renderMVIError()}
         <MultifactorMessage multifactor={multifactor}/>
+        {isBrandConsolidationEnabled &&
+          <div>
```

Comment:
I don't _think_ you need this wrapper `div`

Reply:
The `{ }` has to return a single React node or an array type or it won't compile.

Reply:
or null*